src/Input.js

```diff
@@ -25,7 +25,14 @@ class Input extends Component {
       $(this.selectInput).on('change', this._onChange);
     }
     if (this.isDatePicker) {
-      $(this.dateInput).pickadate(this.props.options);
+      let { options } = this.props;
+      $(this.dateInput).pickadate({...options,
+        onClose: function () {
+          if (options.onClose) {
+            options.onClose();
+          }
+          $(document.activeElement).blur();
```

Comment:
Nice! since this is not fully supported why don't we go with `$(this.dateInput).blur()` ?

Reply:
Okay thats good.

Reply:
`$(this.dateInput).blur()` this is not working because `this.dateInput` is undefined inside onClose function which is key value of an object.